ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
Could you be more specific about what you mean with "is dynamically linked to the RViz process"? Do you mean `move_group` node perhaps? If the latter is the case, shouldn't starting the `move_group` node in a suitable namespace be enough to get it to work with a particular prefix?